Test script and results

RajdeepUppal opened the pull request:
Test script and results for the sprint ending on 18th November 2020

alex-lee-tessella commented on code/workspaces/test/test-results/10893_STS.md:
```diff
@@ -164,16 +178,16 @@ System testing is carried out as part of every sprint to ensure the completed st
+>>> | 4 | Fill in the fields as required and set the data store mount to the pre-existing NFS store. Set the notebook type to Jupyterlab  | NFS store and notebook type is set is set |PASS  |
+>>> | 5 | Click on "Create" | Notebook is created. There is a message in the UI stating that the creation was successful.  | PASS  |
+>>> | 6 | Wait for a minute | THe status is changed from "Requested" to "Ready" | PASS |
+>>> | 7 | Repeat the above tests forpre-existing Gluster store | The same results as teh NFS store (above) are seen | PASS |
```
Missing a space in _forpre-existing_

alex-lee-tessella commented on code/workspaces/test/test-results/10893_STS.md:
```diff
@@ -200,13 +214,25 @@ System testing is carried out as part of every sprint to ensure the completed st
+>>> | **Number** | **Step** | **Expected Result** | **Result** |
+>>> | 1 | Navigate to the notebooks page | Pre-existing notebooks are listed here | PASS |
+>>> | 2 | Click on the create button | A dialog is shown with the following fields: Display Name, Notebook Type, URL, Data Store, Source Path, Description and Sharing Status | PASS |
+>>> | 3 | Fill out the required fields. Set the typ to R studio | And extra field appears when R Studio is selected. This is a version field. 4.0.1 is the default value | PASS |
```
Typo: Set the _typ_ to R studio

alex-lee-tessella commented on code/workspaces/test/test-results/10893_STS.md:
```diff
@@ -200,13 +214,25 @@ System testing is carried out as part of every sprint to ensure the completed st
+>>> | 1 | Navigate to the notebooks page | Pre-existing notebooks are listed here | PASS |
+>>> | 2 | Click on the create button | A dialog is shown with the following fields: Display Name, Notebook Type, URL, Data Store, Source Path, Description and Sharing Status | PASS |
+>>> | 3 | Fill out the required fields. Set the typ to R studio | And extra field appears when R Studio is selected. This is a version field. 4.0.1 is the default value | PASS |
+>>> | 4 | Click on the version field | A dropdpwn with two options is displayed: 4.0.1 & 3.5.3 | PASS |
```
Typo: A _dropdpwn_ with two options

alex-lee-tessella commented on code/workspaces/test/test-results/10893_STS.md:
```diff
@@ -274,7 +300,19 @@ System testing is carried out as part of every sprint to ensure the completed st
 >>> | 6 | Re-add the users email and press save | The user's email is saved | PASS |
 
 ---
+>>>###### **TC2.3.1.6 User can give another user access to storage**
+>>> | **JIRA ID(s):** | [NERCDL-665](https://jira.ceh.ac.uk/browse/NERCDL-665?jql=key%20in%20(NERCDL-665)) |OVERALL RESULT: | |
+>>> | ------------ | --------- | --------- | ------|
+>>> | **Number** | **Step** | **Expected Result** | **Result** |
+>>> | 1 | Ensure that there is at least one other user who has access to the project | There is another user who can view the project |  |
+>>> | 2 | As the project admin, navigate to the project storage page | Projects are listed |  |
+>>> | 3 | Click on the three dotted button and select edit | A dialog is shown with the current storage information |  |
+>>> | 4 | Add a second user to the sotrage and press save | Second user is added |  |
+>>> | 5 | Log in to datalabs as the second user and navigate to the storage page| The project that user was added to is seen here |  |
+>>> | 6 | Log back in as the administrator and remove the second user from the storage | Second user is removed |  |
+>>> | 7 | Log back in as the second user and navigate to the storage page | The storage is no longer listed and the user can no longer access the storage. |  |
```
Is this meant to have the results filled in?

alex-lee-tessella commented on code/workspaces/test/test-results/10893_STS.md:
```diff
@@ -305,6 +343,18 @@ System testing is carried out as part of every sprint to ensure the completed st
+>>> | **Number** | **Step** | **Expected Result** | **Result** |
+>>> | 1 | Navigate to the sites page | Pre-existing sites are listed here | PASS |
+>>> | 2 | Click on the create button | A dialog is shown with the following fields: Display Name, Site Type, URL, Data Store, Source Path, Description and Visibility Status | PASS |
+>>> | 3 | Fill out the required fields. Set the typ to R studio | And extra field appears when R Shiny is selected. This is a version field. 4.0.1 is the default value | PASS |
+>>> | 4 | Click on the version field | A dropdpwn with two options is displayed: 4.0.1 & 3.5.3 | PASS |
```
Same typos in type and dropdown

alex-lee-tessella commented on code/workspaces/test/test-script/10893_STS.md:
```diff
@@ -200,13 +214,25 @@ System testing is carried out as part of every sprint to ensure the completed st
+>>> | **Number** | **Step** | **Expected Result** | **Result** |
+>>> | 1 | Navigate to the notebooks page | Pre-existing notebooks are listed here |  |
+>>> | 2 | Click on the create button | A dialog is shown with the following fields: Display Name, Notebook Type, URL, Data Store, Source Path, Description and Sharing Status |  |
+>>> | 3 | Fill out the required fields. Set the typ to R studio | And extra field appears when R Studio is selected. This is a version field. 4.0.1 is the default value |  |
+>>> | 4 | Click on the version field | A dropdpwn with two options is displayed: 4.0.1 & 3.5.3 |  |
```
I assume the text in here is the same as in the other file so any typos etc. apply to this file too

alex-lee-tessella reviewed (approved):
Couple of typos to sort out, but happy for you to merge it once they have been

andrew-longmore-tessella reviewed (changes requested):
Raj, I think you ought to try adding a spell checker to VS Code, this one looks useful 'Code Spell Checker'.  I'll trial it myself for general use, and add it to the workspace recommendations if I think it's worth it for everyone.

RajdeepUppal commented:
> Raj, I think you ought to try adding a spell checker to VS Code, this one looks useful 'Code Spell Checker'. I'll trial it myself for general use, and add it to the workspace recommendations if I think it's worth it for everyone.

I have added this spell check and corrected spelling issues.
